integration_test/sql/migration.exs

```diff
@@ -65,6 +65,25 @@ defmodule Ecto.Integration.MigrationTest do
     end
```

Comment:
blerg just realized this is the sql migration test and this will only work in pg.